specification/appendix/commitment_discounts.md

```diff
@@ -0,0 +1,278 @@
+# Commitment Discounts
+
+A commitment discount is a billing discount model that offers reduced rates on preselected SKUs in exchange for an obligated usage or spend amount over a predefined term. Commitment discounts typically consist of a set of purchase and usage records within cost and usage datasets.
+
+Usage-based commitment discounts obligate a customer to a predetermined amount of usage over a preselected term, typically measured in "Hours". In some cases, usage-based commitment discounts allow similar resources of different size classifications to relatively benefit from commitment discount rates. This is typically known as *commitment flexibility*, and can be derived by multiplying the number of actual resource hours consumed by a predefined SKU-based coefficient, sometimes referred to as a *normalization factor* or *ratio*, to derive the number of normalized hours consumed.
+
+Spend-based commitment discounts obligate a customer to a predetermined amount of spend over a preselected term, typically denoted by the invoice's selected billing currency (ex: "USD").
+
+## Purchasing
+
+While customers are bound to the term of a commitment discount, cloud-service providers offer some or all of the following payment options before and/or during the term:
+
+* *Upfront* - The commitment discount is paid in full before the term begins.
+* *Recurring* - The commitment discount is paid on a repeated basis, typically over each billing period of the term.
+* *Partial* - Some of the commitment discount is paid before the term begins, and the rest is paid repeatedly over the term.
```

Comment:
Personally, I prefer using the terms All-Upfront, No-Upfront, and Partial-Upfront for payment models to avoid potential confusion with the Recurring Charge Frequency.
As for the descriptions, I suggest we focus on mentioning One-Time and/or Recurring purchases.